server/models/postgis/project.py

```diff
@@ -203,6 +207,17 @@ def update(self, project_dto: ProjectDTO):
             type_array.append(MappingTypes[mapping_type].value)
         self.mapping_types = type_array
 
+        # Cast Editor strings to int array
+        map_editors_array = []
+        for map_editor in project_dto.map_editors:
+            map_editors_array.append(Editors[map_editor].value)
+        self.map_editors = map_editors_array
+
+        validate_editors_array = []
```

Comment:
`mapping_editors_array` and `validation_editors_array`

Reply:
Fixed 23c4bb4